Hunting Queries/CloudAppEvents/SetPolicyConfigInCloudAppEvents.yaml

```diff
@@ -0,0 +1,32 @@
+id: fcd4d774-a0c2-4d12-9e9f-f51dfc310873
+name: 
+description: |
+  "We search for any action type with high frequency that involves adding/modifying/removing something in cloud app policies. We want to see where the properties are modified such that the old value and new value are different, and exclude DisplayName as changing Display Name is a minor property change."

```

Comment:
Lets avoid the team We, I. Refer descriptions from other queries which generally starts with Identifies or This alert will detect etc. I can review this in the chat separately with you for all other queries.